lib/features/poscan_putobject/presentation/widgets/edit_property_button.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:threedpass/core/theme/d3p_colors.dart';
+import 'package:threedpass/features/poscan_putobject/domain/entities/poscan_property.dart';
+
+class EditPropertyButton extends StatelessWidget {
+  final PoscanProperty prop;
+  Function(BuildContext, PoscanProperty) onPressedEditCallback;
```

Comment:
I missed this when I did a review. I think this coulde be declared as `final void Function ...`